### DISCUSSION TITLE
Benefits didn't Received

### DISCUSSION THREAD

**Discussion by Hk-Hacker-Harsh** [community/community#130413] [GitHub Education]
### Hi everyone,

Three days completed since the approval of my student benefit/developer pack, but didn't received benefits yet.
Thank you.

**Comment by davevad93** [comment]
Hi @Hk-Hacker-Harsh  ,

It may take up to a couple of weeks for the benefits to become available, it usually takes just a few days but the waiting time may vary depending on different factors such as the volume of applications.

When the benefits are available the status will change to:

- `Congrats! Your request for @Hk-Hacker-Harsh was approved and your benefits are now available`

For more info see this pinned post: [Student Developer Pack Application FAQs & Common Rejection Reasons (Updated!) 🎒](https://github.com/orgs/community/discussions/111352#top)
